test/k6/src/tests/orders_sms.js

```diff
@@ -122,11 +116,7 @@ function TC03_GetNotificationOrderBySendersReference(data) {
       r.status === 200,
   });
 
-  addErrorCount(success);
-  if (!success) {
-    // only continue to parse and check content if success response code
-    stopIterationOnFail(success);
-  }
+  stopIterationOnFail("GET notification order by senders reference request failed", success);
```

Comment:
not sure if we need to stop the full run if this request fails? Summaries etc. might still run green ?

Reply:
Would it make sense to only have the test that produces the test data stop the full iteration, and for tests that don't have dependences we can let the full test set run ?